server/lib/csp.js

```diff
@@ -13,10 +13,15 @@ var helmet = require('helmet');
 var config = require('./configuration');
 var url = require('url');
 
-var SELF = "'self'";
+var SELF = "'self'"; //jshint ignore: line
```

Comment:
Excellent question, @jrgm! In the past, we've tried to remove the JSHint rule and stick w/ JSCS where possible (since JSHint seems to be removing styling rules).
I say we disable the JSHint rule and let JSCS handle it.

Or, the more dramatic approach, use [**standard**](https://www.npmjs.com/package/standard) instead of **jshint** and **jscs**.